src/Compilers/CSharp/Portable/Parser/SyntaxParser.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax
 {
+    using Microsoft.CodeAnalysis.CSharp.Symbols;
```

Comment:
>Microsoft.CodeAnalysis.CSharp.Symbols [](http://example.com/codeflow?start=10&length=37)

Is this used?

Reply:
nope.  sorry :(  can remove in followup.